<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>custMainPage</class>
 <widget class="QMainWindow" name="custMainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/BG.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>461</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>461</width>
       <height>541</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="productsTab">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <attribute name="title">
       <string>Products</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>461</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="prodTabLay">
        <item>
         <widget class="QTreeWidget" name="prodTree">
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(84, 83, 83);
selection-color: rgb(255, 255, 255);</string>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="background">
            <color>
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButt">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>Add to my basket</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="basketTab">
      <attribute name="title">
       <string>Basket</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>461</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="baskTabLay">
          <item>
           <widget class="QTreeWidget" name="baskTree">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="priceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;</string>
            </property>
            <property name="text">
             <string>Total Price:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buyButt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;</string>
            </property>
            <property name="text">
             <string>Buy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="welcomeLable">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 25 italic 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome user.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
